packages/javalib/javalib.2.3.2/opam

```diff
@@ -0,0 +1,25 @@
+opam-version: "1.2"
+maintainer: "sawja@inria.fr"
+build: [
+  ["./configure.sh"]
+  [make "ptrees"]
+  [make "installptrees"]
+  [make]
+]
+bug-reports: "sawja@inria.fr"
+install: [
+  [make "install"]
+]
+remove: [
+  ["ocamlfind" "remove" "javalib"]
+  ["ocamlfind" "remove" "ptrees"]
+]
+homepage: "http://sawja.inria.fr"
+depends: [
+  "ocamlfind"
+  "camlzip" {= "1.05"}
+  "camlp4" {>= "4.03"}
```

Comment:
The Camlp4 packages are themselves constrained to appropriate OCaml versions, so a simple `"camlp4"` dependency will install a Camlp4 package whose version matches the compiler.